lib/phoenix_mtm/helpers.ex

```diff
@@ -50,6 +52,20 @@ defmodule PhoenixMTM.Helpers do
     )
   end
 
+  defp build_label_with_input(form, field, input_tag, label, label_opts, [nested: true]) do
+    [
+      label form, field, label_opts do
+        [{:safe, "#{label}"}, input_tag]
```

Comment:
On line 49 below, the entire group is escaped so I think this should be safe.

I like having the escape as the "final" call that way we can't possibly miss one depending on which fork in the road is chosen.